docs/core/diagnostics/available-counters.md

```diff
@@ -78,3 +78,62 @@ The following counters are published as part of the [ASP.NET Core Kestrel web se
 | :::no-loc text="Total TLS Handshakes"::: (`total-tls-handshakes`) | The total number of TLS handshakes with the web server |
+
+## "System.Net.Http" counters (available on .NET 5 and later versions)
+
+The following counters are published by the HTTP stack.
+
+| Counter | Description |
+|--|--|
+| :::no-loc text="Requests Started"::: (`requests-started`) | The number of requests started since the process started |
+| :::no-loc text="Requests Started Rate"::: (`requests-started-rate`) | The number of requests started per update interval |
+| :::no-loc text="Requests Failed"::: (`requests-failed`) | The number of requests failed since the process started |
+| :::no-loc text="Requests Failed Rate"::: (`requests-failed-rate`) | The number of requests failed per update interval |
+| :::no-loc text="Current Requests"::: (`current-requests`) | Current number of active HTTP requests that have started but not yet completed or failed |
+| :::no-loc text="Current HTTP 1.1 Connections"::: (`http11-connections-current-total`) | The current number of HTTP 1.1 connections that have started but not yet completed or failed.|
+| :::no-loc text="Current HTTP 2.0 Connections"::: (`http20-connections-current-total`) | The current number of HTTP 2.0 connections that have started but not yet completed or failed.|
+| :::no-loc text="HTTP 1.1 Requests Queue Duration"::: (`http11-requests-queue-duration`) | The average duration of the time HTTP 1.1 requests spent in the request queue |
+| :::no-loc text="HTTP 2.0 Requests Queue Duration"::: (`http20-requests-queue-duration`) | The average duration of the time HTTP 2.0 requests spent in the request queue |
+
+## "System.Net.NameResolution" counters (available on .NET 5 and later versions)
+
+The following counters track metrics related to DNS lookups.
+
+| Counter | Description |
+|--|--|
+| :::no-loc text="DNS Lookups Requested"::: (`dns-lookups-requested`) | The number of DNS lookups requested since the process started |
+| :::no-loc text="Average DNS Lookup Duration"::: (`dns-lookups-duration`) | The average time taken for DNS lookups |
+
+## "System.Net.Security" counters (available on .NET 5 and later versions)
+
+The following counters track metrics related to the Transport Layer Security protocol.
+
+| Counter | Description |
+|--|--|
+| :::no-loc text="TLS handshakes completed"::: (`tls-handshake-rate`) | The number of TLS handshakes completed per update interval |
+| :::no-loc text="Total TLS handshakes completed"::: (`total-tls-handshakes`) | The total number of TLS handshakes completed since the process started |
+| :::no-loc text="Current TLS handshakes"::: (`current-tls-handshakes`) | The current number of TLS handshakes that have started but not yet completed |
+| :::no-loc text="Total TLS handshakes failed"::: (`failed-tls-handshakes`) | The total number of TLS handshakes failed since the process started |
```

Comment:
```suggestion
| :::no-loc text="Total TLS handshakes failed"::: (`failed-tls-handshakes`) | The total number of failed TLS handshakes since the process started |
```